src/vs/workbench/contrib/chat/browser/chatInputPart.ts

```diff
@@ -465,7 +465,7 @@ export class ChatInputPart extends Disposable implements IHistoryNavigationWidge
 			return;
 		}
 
-		mode = validateChatMode(mode) ?? ChatMode.Ask;
+		mode = validateChatMode(mode) ?? (this.location === ChatAgentLocation.Panel ? ChatMode.Ask : ChatMode.Edit);
```

Comment:
Is this an intentional change for this PR?

Reply:
Yes, this avoids the edits view (if unified view is disabled) setting itself to Ask mode. Not strictly part of the welcome view change but I came across it while testing this with and without unified mode.